android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -674,12 +674,18 @@
 
 	/**
 	 * @module.api
+	 * @deprecated as of Android 6.0 this property will be replaced by PROPERTY_ENTER_TRANSITION
+	 * and will be removed in android 7.0
 	 */
+	@Deprecated
```

Comment:
Also add the docs so people actually now about the deprecation. Adding a `@deprecated` annotation does not automatically throw a warning, so they should have the docs to lookup.